airbyte-json-validation/src/main/java/io/airbyte/validation/json/JsonSchemaValidator.java

```diff
@@ -52,7 +53,15 @@ public JsonSchemaValidator() {
     this.jsonSchemaFactory = JsonSchemaFactory.getInstance(SpecVersion.VersionFlag.V7);
   }
 
-  public Set<ValidationMessage> validate(JsonNode schemaJson, JsonNode objectJson) {
+  public Set<String> validate(JsonNode schemaJson, JsonNode objectJson) {
+    return validateInternal(schemaJson, objectJson)
+        .stream()
+        .map(ValidationMessage::getMessage)
+        .collect(Collectors.toSet());
+  }
+
+  // keep this internal as it returns a type specific to the wrapped library.
```

Comment:
just trying to avoid packages that depend on this one needing to all add networknt validation as a dependency too.

Reply:
Now that you've created the `airbyte-json-validation/src/main/java/io/airbyte/validation/json/ConfigJsonSchemaValidator.java`, I think you should kill all the public methods but `Set<String> validate`, and move all the addional behaviors in ConfigJsonSchemaValidator instead.

Reply:
What's the logic behind that? We have use cases that use the public methods on the core validator. For example, since we are using it to validate configs using the specs returned by connectors, so we cannot leverage the new abstraction in those cases.